### ISSUE TITLE
[1.10.2] Conveyor Belt Chunk Loading Issues

### ISSUE THREAD

**Issue by oitsjustjose** [BluSunrize/ImmersiveEngineering#2116]
## The Bug:

This bug is... odd. And very specific, but not in the sense of "edge case" specific. I couldn't very well describe this in words, so I made a [video demonstration](https://youtu.be/3tUJZuhi2qE) instead.

A breakdown of what happens is that, when I log off of my dedicated server, the garden cloches will continue to run. Instead of outputting to the conveyor belts like they _should_ and _do_ when I'm logged on, they clog up and bring the server to its knees (and literally cause a crash due to being unable to write so many entities to the chunk data).

## Additional Info:

To be clear: the area is being completely chunk-loaded by Persistent Bits chunk-loaders. These cover this area just fine ~~and the problem doesn't happen with _all_ of the garden cloches, just these at this specific "floor" of my gardens.~~

I'd just use EnderIO conduits, but they don't seem to extract even though they're displaying as connected. Connecting an adjacent inventory works fine, but is.... clunky and ugly.

_____

Mod Info:
* Forge 12.18.3.2281
* Immersive Engineering 0.10-58
* [Full Modlist](http://puu.sh/vYCjN/7af4a39a5f.txt)

_____

### Let me know how else I can help!!

**Comment by BluSunrize** [comment]
That's an interesting one for sure.
First of all, what's the mod that makes items lie down? It's the same one as in the Invasion pack, where you need to rightclick to pick them up, correct? Can you test without that and see if it causes the issue?

**Comment by malte0811** [comment]
RealisticItemDrops. I suspected that as well, but it looks like it should work from a quick glance over the code. Testing without it might be a good idea anyways.

**Comment by oitsjustjose** [comment]
Confirmed - I removed RealisticItemDrops and the problem persists.

**Comment by MalkContent** [comment]
Is it possible that some of those are redstone ignoring conveyors (possibly even created by accident, see https://github.com/BluSunrize/ImmersiveEngineering/issues/2109)?
Y'know, just making sure that the setup for the up-going/down-going  setup is not only visually identical.

**Comment by oitsjustjose** [comment]
Very interesting... I'll take a look when I get home from work!

**Comment by oitsjustjose** [comment]
Update (updating the original request too): Bottom conveyors also lagging out - so it's at least a consistent conveyor mis-behavior.

**Comment by MalkContent** [comment]
Idk, I don't think redstone ignoring conveyors should cause it, I'm just making sure its covered anyways ^^

**Comment by oitsjustjose** [comment]
Yeah, just double checked - everything's as expected as far as conveyor types go.

**Comment by oitsjustjose** [comment]
Changing the title because I've spent a few hours debugging this and have come to a few discoveries:

1. Placing a chunk loader (my own or ExU2's) in the **exact** chunk which the farm is set up works fine.
2. **Not** placing a chunk loader in the **exact** chunk, but having other machines (I used a creative drawer sending items in to a normal drawer as a measurement / test to confirm the chunks _were_ actually loaded) works just fine with everything else, except for conveyor belts on cloches.

At first I was seriously sweating that it was my Chunk Loader code, but I was able to repeat the same exact results by using Extra Utilities' Chunk Loading Ward, so I _think_ it's safe to say it's not me. Did I read somewhere that all of IE no longer uses ticking tile entities? If so, this might be a result / side-effect, but I'm not really certain.... I do a wide variety of programming, but some things in Minecraft's code base just don't click to me such as advanced Tile Entity logic and pretty much any TESR / Rendering code.

**Comment by oitsjustjose** [comment]
Alright, so I tried it in my Singleplayer Let's Play world (identical setup) and had no problems even in the damn end. So - I was curious; I copied my PersistentBits jar, and its config file along with the forge and forge chunkloading configs (NOT IE's though), uploaded them to my server (and I can't tell a difference in the settings I had chosen...) and now everything's working fine.

No longer can reproduce. Closing for now.
